libs/ledger-live-common/src/families/solana/api/chain/web3.ts

```diff
@@ -271,6 +272,36 @@ export async function getStakeAccountAddressWithSeed({
   return pubkey.toBase58();
 }
 
+export async function getPriorityFee(api: ChainAPI, accounts: PublicKey[]): Promise<number> {
+  const uniqAccs = uniqBy(accounts, acc => acc.toBase58());
+  const recentFees = await api.getRecentPrioritizationFees({
+    lockedWritableAccounts: uniqAccs,
+  });
+
+  return median(recentFees.map(item => item.prioritizationFee));
```

Comment:
I am no solana expert :sweat_smile: Would using lowest value work ?

Reply:
Hi Hedi
Good question :)
I've been thinking about this as well, and after research it looked like median seem be good to approximate priority fees for a given transaction. This ensures that the priority fees is above the lowest but not too wasteful.

Helius Priority Fee API also uses 50th percentile (median) by default.
But it also provides other values, like low (25th) or UNSAFE_MAX (100th).

Reply:
Ok thanks !